### DISCUSSION TITLE
issue: Cannot access into any component wrapped by Controller using react-hook-form

### DISCUSSION THREAD

**Discussion by vadimlab** [react-hook-form/react-hook-form#9783] [Q&A]
### Version Number

7.29.0

### Codesandbox/Expo snack

N/A

### Steps to reproduce

1.  Build ios app `ipa` and upload it to `browserstack` 
 
![image](https://user-images.githubusercontent.com/108320644/212629112-4902a023-238b-4f0e-b31b-bb2368b8aac3.png)

As you see, I can't access child component of the component which wrapped by `Controller` from `react-hook-form`

### Expected behaviour

Can access child component so each test label should be seperated

### What browsers are you seeing the problem on?

Chrome

### Relevant log output

```shell
![image](https://user-images.githubusercontent.com/108320644/212629112-4902a023-238b-4f0e-b31b-bb2368b8aac3.png)
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by vadimlab** [comment]
Please help me this issue

**Comment by leapful** [comment]
@vadimlab 

This is not an issue from react-hook-form but from BrowserStack inspector dev tool.

Based on your information, it might be related to max depth of snapshot: https://github.com/appium/appium/issues/14825#issuecomment-720482679

Please try to increase your snapshot max depth number limit.

**Comment by vadimlab** [comment]
@leapful it's not the issue, if I don't use `react-hook-form` controller, it shows child accessibility correctly.

**Comment by leapful** [comment]
@vadimlab 

Could you please provide codesandbox or github repo to re-produce the problem ?

**Comment by vadimlab** [comment]
@leapful here is example of the issue happened
`<ScrollView keyboardShouldPersistTaps={'handled'}>
  <View>
  <Controller
            control={control}
            name="overview"
            render={({ field, fieldState }) => (
              <TextInput
                label="Bio"
                labelTextStyle={H6Bold}
                labelStyle={styles.labelStyle}
                onChangeText={field.onChange}
                onBlur={field.onBlur}
                text={field.value ?? ''}
                multiline
                textInputStyle={styles.bioText}
                textContainerStyle={styles.bio}
                containerStyle={styles.textContainer}
                error={fieldState.error?.message}
                maxLength={OVERVIEW_LIMIT}
              />
            )}
          />
  </View>
</ScrollView>`

![image](https://user-images.githubusercontent.com/108320644/213101127-b6eb5b9f-9ed7-4248-b348-38ab8c4ee702.png)
as you see screenshot, can't get access into `TextInput` as it wrapped by `Controller`
